app/src/pages/playground/playgroundUtils.ts

```diff
@@ -230,7 +230,26 @@ export function getTemplateMessagesFromAttributes({
       messages: null,
     };
   }
-
+  if (provider === "ANTHROPIC") {
```

Comment:
if semantic conventions are somewhat provider agnostic, I think it is fine to perform this logic anytime we see `llm.invocation_parameters.system`.

For example, gemini does not support system role messages, instead using a top level `system_instruction` field. So it seems likely we may want to update our playground instrumentation to record system messages as llm.invocation_parameters.system if it does not already, making this code also relevant for gemini.

Reply:
The gemini `system_instruction` is a list of parts (see below). So I'm just keep this ad hoc for the time being.

<img width="301" alt="Screenshot 2024-11-26 at 8 14 00 AM" src="https://github.com/user-attachments/assets/24be51a9-8a40-44d6-be55-6e1dbac04077">